### ISSUE TITLE
Make Failed

### ISSUE THREAD

**Issue by Arvinzhu** [arisada/midgetpack#6]
OS: Ubuntu 18.04 LTS
gcc version 7.3.0 (Ubuntu 7.3.0-27ubuntu1~18.04) 

```
[ 37%] Linking ASM-ATT executable stub_freebsd_amd64
/usr/bin/ld: cannot find -lgcc_s
/usr/bin/ld: cannot find -lgcc_s
src/stub/CMakeFiles/stub_freebsd_amd64.dir/build.make:106: recipe for target 'src/stub/stub_freebsd_amd64' failed
make[2]: *** [src/stub/stub_freebsd_amd64] Error 1
CMakeFiles/Makefile2:486: recipe for target 'src/stub/CMakeFiles/stub_freebsd_amd64.dir/all' failed
make[1]: *** [src/stub/CMakeFiles/stub_freebsd_amd64.dir/all] Error 2
Makefile:83: recipe for target 'all' failed
make: *** [all] Error 2
```

Could someone tell me how to resolve  this problem  ?

**Comment by Arvinzhu** [comment]
This problem maybe cased by the version of CMake.Version 3.7.2  could  make sucessfully.

**Comment by oakieville** [comment]
i have same error tried many versions cmake all same result
is possible anyone has precompiled midgetpack i can use?

edit: i now can compile it howevr after packing binary binary no longer runs 
syntax error: unexpected '?⌠4╬█KE+Ω⌡k'

**Comment by 15611600466** [comment]
> This problem maybe cased by the version of CMake.Version 3.7.2 could make sucessfully.

Yes!! https://github.com/Kitware/CMake/releases/download/v3.7.2/cmake-3.7.2-Linux-x86_64.tar.gz

cmake-3.7.2-Linux-x86_64/bin/cmake ..

Scanning dependencies of target mp_shared_static_amd64
[  2%] Building C object src/shared/CMakeFiles/mp_shared_static_amd64.dir/aes.c.o
[  5%] Building C object src/shared/CMakeFiles/mp_shared_static_amd64.dir/crypto.c.o
[  7%] Building C object src/shared/CMakeFiles/mp_shared_static_amd64.dir/curve25519_ref.c.o
[ 10%] Building C object src/shared/CMakeFiles/mp_shared_static_amd64.dir/sha256.c.o
[ 13%] Linking C static library libshared_amd64.a
[ 13%] Built target mp_shared_static_amd64
Scanning dependencies of target mp_shared_static
[ 15%] Building C object src/shared/CMakeFiles/mp_shared_static.dir/aes.c.o
[ 18%] Building C object src/shared/CMakeFiles/mp_shared_static.dir/crypto.c.o
[ 21%] Building C object src/shared/CMakeFiles/mp_shared_static.dir/curve25519_ref.c.o
[ 23%] Building C object src/shared/CMakeFiles/mp_shared_static.dir/sha256.c.o
[ 26%] Linking C static library libshared.a
[ 26%] Built target mp_shared_static
Scanning dependencies of target mp_libstub_static_amd64
[ 28%] Building C object src/stub/CMakeFiles/mp_libstub_static_amd64.dir/pack_common.c.o
[ 31%] Linking C static library libstubamd64.a
[ 31%] Built target mp_libstub_static_amd64
Scanning dependencies of target stub_freebsd_amd64
[ 34%] Building ASM-ATT object src/stub/CMakeFiles/stub_freebsd_amd64.dir/pack_amd64.s.o
[ 36%] Building ASM-ATT object src/stub/CMakeFiles/stub_freebsd_amd64.dir/freebsd_amd64.s.o
[ 39%] Linking ASM-ATT executable stub_freebsd_amd64
[ 39%] Built target stub_freebsd_amd64
Scanning dependencies of target stub_linux_amd64
[ 42%] Building ASM-ATT object src/stub/CMakeFiles/stub_linux_amd64.dir/pack_amd64.s.o
[ 44%] Building ASM-ATT object src/stub/CMakeFiles/stub_linux_amd64.dir/linux_amd64.s.o
[ 47%] Linking ASM-ATT executable stub_linux_amd64
[ 47%] Built target stub_linux_amd64
[ 50%] Generating stub_linux_x86
[ 52%] Generating stub_linux_armv6
[ 55%] Generating stub_linux_armv6.o
[ 57%] Generating stub_freebsd_amd64.o
[ 60%] Generating stub_linux_amd64.o
[ 63%] Generating stub_freebsd_x86
[ 65%] Generating stub_freebsd_x86.o
[ 68%] Generating stub_linux_x86.o
Scanning dependencies of target stubs
[ 71%] Linking C static library libstubs.a
[ 71%] Built target stubs
Scanning dependencies of target mpkex
[ 73%] Building C object src/packer/CMakeFiles/mpkex.dir/crypto.c.o
[ 76%] Building C object src/packer/CMakeFiles/mpkex.dir/mpkex.c.o
[ 78%] Linking C executable mpkex
[ 78%] Built target mpkex
Scanning dependencies of target midgetpack
[ 81%] Building C object src/packer/CMakeFiles/midgetpack.dir/crypto.c.o
[ 84%] Building C object src/packer/CMakeFiles/midgetpack.dir/elf.c.o
[ 86%] Building C object src/packer/CMakeFiles/midgetpack.dir/midgetpack.c.o
[ 89%] Linking C executable midgetpack
[ 89%] Built target midgetpack
Scanning dependencies of target test-amd64-dynamic
[ 92%] Building C object src/tests/CMakeFiles/test-amd64-dynamic.dir/test.c.o
[ 94%] Linking C executable test-amd64-dynamic
[ 94%] Built target test-amd64-dynamic
Scanning dependencies of target test-amd64-static
[ 97%] Building C object src/tests/CMakeFiles/test-amd64-static.dir/test.c.o
[100%] Linking C executable test-amd64-static
[100%] Built target test-amd64-static
